File: Assignment 2/Program1.c
#include<stdio.h>

void Display(int iNo)
{
    int iCnt=0;
    if(iNo<0)
    {
        iNo=-iNo;
    }

    while(iCnt<iNo)
    {
        printf("*\t");
        iCnt++;
    }


}
int main()
{
    int iValue=0;

    printf("Enter Number of Stars:");
    scanf("%d",&iValue);

    Display(iValue);

    return 0;
}

File: Assignment 26/Program1.java
import java.util.*;

class Pattern
{
    public void Display(int iNo)
    {
        int iCnt=0;
        char ch='A';

        for(iCnt=1;iCnt<=iNo;iCnt++)
        {
            System.out.print(ch+"\t");
            ch++;
            if(ch>'Z')
            {
                ch='A';
            }
        }

    }
}
class Program1
{
    public static void main(String A[])
    {
        int iValue=0;

        Scanner sobj=new Scanner(System.in);
        System.out.println("Enter the number:");

        iValue=sobj.nextInt();

        Pattern pobj=new Pattern();
        pobj.Display(iValue);
    }
}